starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/ItemsObject.java

```diff
@@ -0,0 +1,49 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by jay on 10/25/16. */
+
+public class ItemsObject {
+    String mItemName, mDescription, mType;
+    String mPrice;
```

Comment:
It would be a good idea to make these variables `private` in order to restrict the ability to modify them.